spec/grocer_spec.rb

```diff
@@ -251,11 +252,11 @@
       it "calls on #apply_clearance after calling on #apply_coupons with multiple items and one coupon" do
         beer = find_item_by_name_in_collection('BEER', items)
         cart = [find_item_by_name_in_collection('BEETS', items), beer, beer, beer]
-        c = [coupons[1]]
+        coupon = [coupons[1]]
```

Comment:
Not accurate. This is an array. Calling it coupons clashes with the let() statement up top (bitter, bitter experience)

Reply:
oh my mistake. It can go back, I just wanted something there that was spelled out